Compilation fails on 2.4.1

I try to compile stratagus 2.4.1 on opensuse leap 42.2. The compilation breaks at this point:

```
In file included from /home/comrad/Downloads/stratagus-2.4.1/src/ai/ai_plan.cpp:36:0:
/home/comrad/Downloads/stratagus-2.4.1/src/ai/ai_plan.cpp: In member function ‘int AiForce::PlanAttack()’:
/home/comrad/Downloads/stratagus-2.4.1/src/ai/ai_plan.cpp:346:13: error: unable to find string literal operator ‘operator"" _C_’
  DebugPrint("%d: Planning for force #%lu of player #%d\n"_C_ player.Index
             ^
/home/comrad/Downloads/stratagus-2.4.1/src/include/stratagus.h:126:62: note: in definition of macro ‘DebugPrint’
  do { if (EnableDebugPrint) { PrintFunction(); PrintOnStdOut(args); } } while (0)
                                                              ^
CMakeFiles/stratagus.dir/build.make:1169: recipe for target 'CMakeFiles/stratagus.dir/src/ai/ai_plan.cpp.o' failed
make[2]: *** [CMakeFiles/stratagus.dir/src/ai/ai_plan.cpp.o] Error 1
CMakeFiles/Makefile2:104: recipe for target 'CMakeFiles/stratagus.dir/all' failed
make[1]: *** [CMakeFiles/stratagus.dir/all] Error 2
Makefile:127: recipe for target 'all' failed
make: *** [all] Error 2
```

Any ideas what could cause this?

--- a/src/ai/ai_plan.cpp
+++ b/src/ai/ai_plan.cpp
@@ -343,7 +343,7 @@ int GetTotalBoardCapacity(ITERATOR begin, ITERATOR end)
 int AiForce::PlanAttack()
 {
 	CPlayer &player = *AiPlayer->Player;
-	DebugPrint("%d: Planning for force #%lu of player #%d\n"_C_ player.Index
+	DebugPrint("%d: Planning for force #%lu of player #%d\n" _C_ player.Index
 			   _C_(long unsigned int)(this - & (AiPlayer->Force[0])) _C_ player.Index);
 
 	TerrainTraversal transporterTerrainTraversal;

--- a/src/stratagus/script.cpp
+++ b/src/stratagus/script.cpp
@@ -767,7 +767,7 @@ NumberDesc *CclParseNumberDesc(lua_State *l)
 			lua_pop(l, 1); // table.
 		} else {
 			lua_pop(l, 1);
-			LuaError(l, "unknow condition '%s'"_C_ key);
+			LuaError(l, "unknow condition '%s'" _C_ key);
 		}
 	} else {
 		LuaError(l, "Parse Error in ParseNumber");
@@ -881,7 +881,7 @@ StringDesc *CclParseStringDesc(lua_State *l)
 			res->D.PlayerName = CclParseNumberDesc(l);
 		} else {
 			lua_pop(l, 1);
-			LuaError(l, "unknow condition '%s'"_C_ key);
+			LuaError(l, "unknow condition '%s'" _C_ key);
 		}
 	} else {
 		LuaError(l, "Parse Error in ParseString");

